build/source/driver/summa_alarms.f90

```diff
@@ -86,7 +86,7 @@ subroutine summa_setWriteAlarms(oldTime, newTime, endTime,       &   ! time vect
  character(*),intent(out)              :: message            ! error message
  ! ---------------------------------------------------------------------------------------
  ! local variables
- character(LEN=256)                    :: cmessage           ! error message of downwind routine
+ !character(LEN=256)                    :: cmessage           ! error message of downwind routine
```

Comment:
never used - delete

Reply:
Not really 'never used' ... rather it's good to see this PR's restructuring of summa control files and other usability changes (eg writing GRU ids) merged into the master via your subsequent split PRs (leading in part to version 3.0).  

It looks like there may be one discrepancy with this original that's now an error the current code -- nf90_def_var(ncid, 'gruId', nf90_int64, gru_DimID, gruIdVarID); -- is now writing 'nf90_int' -- I'm not sure where this change happened, since the newer version worked fine up to now (v3.0.2).  Trying to track it ...

Reply:
hopefully this is not me looking at a file from the wrong branch